README.md

```diff
@@ -4,6 +4,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm install`
+
+Installs all necessary deppendencies listed in `package.json`.
+
 ### `npm start`
```

Comment:
You may want to change this to `yarn start`, which will work as well and is more consistent with the yarn setup.